css/styles.css

```diff
@@ -55,39 +60,307 @@ a {
 	clear: both;
 }
 
+/* Photodiary */
+
+.photo-diary {
+	width: 85%;
+	padding-bottom: 100px;
+	margin: 0 auto;
+	color: rgba(115, 115, 115, .9);
+}
+
+.photo-diary span {
+	display: block;
+	font-size: 0.9rem;
+	font-family: 'Ubuntu', sans-serif;
+	text-transform: uppercase;
+	opacity: .6;
+}
+
+.photo-diary h1 {
+	padding-bottom: 10px;
+	font-family: "Playfair Display", serif;
+	font-weight: normal;
+}
+
+.photo-diary p {
+	display: block;
+	padding-bottom: 40px;
+	font-family: "Playfair Display", serif;
+	font-size: 0.9rem;
+}
+
+
+/* Div with 4 articles */
+
+.articles-div {
+	width: 85%;
+	padding-top: 80px;
+	padding-bottom: 40px;
+	margin: 0 auto;
+	color: rgba(115, 115, 115, .9);
+}
+
+.article img {
+	width: 90%;
+	padding-bottom: 30px;
+}
+
+.article span {
+	display: block;
+	font-family: 'Ubuntu', sans-serif;
+	font-size: 0.9rem;
+	text-transform: uppercase;
+	opacity: .6;
+}
+
+.article h1 {
+	font-family: "Playfair Display", serif;
+	font-weight: normal;	
+}
+
+.article p {
+	display: block;
+	padding-bottom: 30px;
+	font-family: "Playfair Display", serif;
+	font-size: 0.9rem;
+}
+
+
+/* Newsletter sign-up */
+
+.newsletter-div {
+	position: relative;
+	height: 260px;
+	background: #f2f2f2;
+	font-family: "Playfair Display", serif;
+	text-align: center;
+}
+
+.newsletter-div legend {
+	padding-top: 50px;
+	margin: 0 auto;
+	font-size: 1.4rem;
+}
+
+.newsletter-div input {
+	position: absolute;
+	top: 50%;
+	right: 20%;
+	display: inline-block;
+	width: 60%;
+	height: 30px;
+	border: none;
+	border-bottom: 2px solid #000;
+	background: #f2f2f2 url("../images/Send_icon.png") no-repeat right / 8%;
+	color: #000;
+	font-size: 0.8rem;
+}
+
+.newsletter-div input:focus {
+	outline: none;
+}
+
+
+.newsletter-div input::-webkit-input-placeholder {
+	color: #000;	
+}
+
+
+/* Button load-more */
+
+.btn-div {
+	padding: 80px 0;
+}
+
+.btn {
+	display: block;
+	width: 180px;
+	height: 60px;
+	padding: 10px 0;
+	border: 1px solid ;
+	margin: 0 auto;
+	font: 1.4rem "Playfair Display", serfif;
+	text-align: center;
+}
+
+ .btn-div a {
+ 		display: block;
+ }
+

```

Comment:
Już pamiętam czemu tak zrobiłem - miałem problem z zapanowaniem nad tekstem "Load more". W tej chwili pozbyłem się paddingów i tekst nie jest wycentrowany wertykalnie. 

Próbowałem dać mu `display: inline-block` i `vertical-align: middle` ale nic z tego. Mógłbyś mi podpowiedzieć jak najlepiej to zrobić?